ISICbalance/FlowCoordinators/AppFlowCoordinator.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  AppFlowCoordinator.swift
+//  ISICbalance
+//
+//  Created by Rostislav Babáček on 05/05/2019.
+//  Copyright © 2019 Rostislav Babáček. All rights reserved.
+//
+
+import UIKit
+
+class AppFlowCoordinator: BaseFlowCoordinator {
+    public var childCoordinators = [BaseFlowCoordinator]()
+
+    weak var navigationController: UINavigationController!
+
+    func start(in window: UIWindow) {
+        let navigationController = UINavigationController()
+        window.rootViewController = navigationController
+        self.navigationController = navigationController
+
+        let balanceVM = BalanceViewModel(dependencies: AppDependency.shared)
+        let balanceVC = BalanceViewController(viewModel: balanceVM)
+        balanceVC.flowDelegate = self
+        navigationController.setViewControllers([balanceVC], animated: true)
```

Comment:
prohodil bych vytvoření VC a VM s vytvořením navigation controlleru. Pak můžeš zavolat `let navigationController = UINavigationController(rootViewController: balanceVC)` místo nastavování controllerů napřímo

Reply:
Jo to dává smysl no. Já tam první chtěl první udělat nějaký if podle výsledku requestu, ale nedává to moc smysl.